modules/aws/auth.go

```diff
@@ -13,8 +14,18 @@ import (
 	"github.com/pquerna/otp/totp"
 )
 
-// NewAuthenticatedSession gets an AWS Session, checking that the user has credentials properly configured in their environment.
+// NewAuthenticatedSession creates an AWS session following to standard AWS authentication workflow.
+// If TERRATEST_IAM_ROLE environment variable is set, assumes IAM role specified in it.
 func NewAuthenticatedSession(region string) (*session.Session, error) {
+	if assumeRoleArn, ok := os.LookupEnv("TERRATEST_IAM_ROLE"); ok {
```

Comment:
NIT: Can you extract this string constant to a global `const` at the top of the file so it is easier to change later, even if the file evolves overtime?

Reply:
good point, fixed